contracts/Sale.sol

```diff
@@ -216,6 +222,15 @@ contract Sale {
         owner = _newOwner;
     }
 
+    function withdrawRemainder()
+        onlyOwner
+        saleStarted
+        saleEnded
+    {
+        uint remainder = token.balanceOf(this);
+        token.transfer(wallet, remainder);
```

Comment:
Maybe this should take an argument for the transfer address. If a token seller uses, like, a Coinbase address for their wallet (they shouldn't, but...) and withdraws to it they'll lose their tokens.

Reply:
Open to discussion on this though.

Reply:
I was thinking of that, but having still another wallet address in the config parameters is also not optimal...

Reply:
You (the owner) would just pass it in as an argument to _this function_. No need for it in the constructor methinks.

Reply:
Ahhh, sure! 👍